conf/application.conf

```diff
@@ -1,5 +1,20 @@
-play.server.http.port = disabled
-play.server.https.port = 9443
+play {
+  server {
+    http.address = localhost
+    http.address = ${?RP_ENDPOINT_0_BIND_HOST}
+    http.port = 9000
+    http.port = ${?RP_ENDPOINT_0_BIND_PORT}
+    https.address = localhost
+    https.address = ${?RP_ENDPOINT_1_BIND_HOST}
+    https.port = 9443
+    https.port = ${?RP_ENDPOINT_1_BIND_PORT}
```

Comment:
the RP prefix comes from the `rp` tool. I don't think we should carry this over. It's not clear why we are prefixing it with RP and it doesn't add any extra value.